### PR TITLE
Improved look for modals

### PR DESCRIPTION
**Pull request by danlugo92**
Before / after-

Screen Shot 2021-08-05 at 4 49 12 PM

Screen Shot 2021-08-05 at 4 58 11 PM

Screen Shot 2021-08-05 at 5 15 41 PM

Screen Shot 2021-08-05 at 5 05 38 PM

Screen Shot 2021-08-05 at 5 19 55 PM

Screen Shot 2021-08-05 at 5 19 07 PM

Screen Shot 2021-08-05 at 5 26 10 PM

Screen Shot 2021-08-05 at 5 25 31 PM
------
IMAGE 2021-08-05 17:32:17
IMAGE 2021-08-05 17:32:25

IMAGE 2021-08-05 17:32:39

IMAGE 2021-08-05 17:32:44

IMAGE 2021-08-05 17:32:51

IMAGE 2021-08-05 17:32:57

IMAGE 2021-08-05 17:33:02

IMAGE 2021-08-05 17:33:06

-----------------------------------------------------------------------------

Screenshot from 2021-08-07 14-00-17

Screenshot from 2021-08-07 14-11-17

photo_2021-08-07_14-36-14

photo_2021-08-07_14-36-19

----------------------------------------------------

Screenshot from 2021-08-07 14-55-50

Screenshot from 2021-08-07 14-56-26

Screenshot from 2021-08-07 14-57-04

Screenshot from 2021-08-07 14-57-18

Screenshot from 2021-08-07 15-09-36

Screenshot from 2021-08-09 13-15-05

image

image
------------
Screen Shot 2021-08-10 at 11 20 38 AM

Screen Shot 2021-08-10 at 11 21 28 AM
-----------
Screen Shot 2021-08-10 at 1 42 12 PM
 
Screen Shot 2021-08-10 at 1 46 35 PM
-----------------
Screen Shot 2021-08-10 at 1 49 03 PM
Screen Shot 2021-08-10 at 2 32 00 PM

----------------------
Screen Shot 2021-08-10 at 3 02 03 PM

Screen Shot 2021-08-10 at 3 01 18 PM

### CODE REVIEW

File: src/common/Modal/components/ModalContent/index.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-// @ts-check
+import c from "classnames";
```
Comment (Emad-salah) [inline src/common/Modal/components/ModalContent/index.js:1]: Can you modify the variable's name so it's more descriptive (ex: `classNames`)?

File: src/common/Modal/css/index.scoped.css [hunk 2]
```diff
@@ -120,7 +120,6 @@
   background-color: var(--failure-red-hovered);
 }
 
-.form .form-error {
-  margin: 0 25px;
-  margin-bottom: 5px;
+.form-error-margin {
+  margin-bottom: 0em !important;
```
Comment (Emad-salah) [inline src/common/Modal/css/index.scoped.css:124]: `margin-bottom: 0 !important;`

**Review by Emad-salah — CHANGES_REQUESTED** [review]

File: src/pages/Profile/PublishContent.tsx [hunk 3]
```diff
@@ -543,23 +543,18 @@ const PublishContentPage = () => {
...
-              onClick={submitPrompt}
-            >
-              confirm
-            </button>
-          </div>
+        <Modal
+          modalOpen={promptInfo && !loading}
+          toggleModal={closePrompt}
+          contentClass="p-1"
+          forceRenderTitleBar
```
[the remark this answers is not in the document]
Comment (danlugo92) [reply src/pages/Profile/PublishContent.tsx:550]: tomahto Tomato

File: src/pages/Advanced/components/AddChannelModal/index.js [hunk 4]
```diff
@@ -2,15 +2,16 @@ import React, { useCallback, useState } from "react";
...
+import { useDispatch } from "react-redux";
 import Loader from "../../../../common/Loader";
+import Pad from "../../../../common/Pad";
 import { loadFeeRates } from "../../../../actions/FeesActions";
 
 const AddChannelModal = ({ open = false, toggleModal }) => {
   const dispatch = useDispatch();
   const [loading, setLoading] = useState(false);
-  const [channelCapacity, setChannelCapacity] = useState("");
+  const [channelCapacity, setChannelCapacity] = React.useState("");
```
[the remark this answers is not in the document]
Comment (danlugo92) [reply src/pages/Advanced/components/AddChannelModal/index.js:14]: Fixed

File: src/common/Modal/components/ModalContent/index.js (hunk 1 above)
Comment (danlugo92) [reply src/common/Modal/components/ModalContent/index.js]: Fixed.

File: src/common/Modal/css/index.scoped.css (hunk 2 above)
Comment (danlugo92) [reply src/common/Modal/css/index.scoped.css:124]: done.

Comment (danlugo92) [reply src/common/Modal/css/index.scoped.css:124]: done.Done.

File: src/pages/Profile/PublishContent.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (danlugo92) [reply src/pages/Profile/PublishContent.tsx:550]: Added to TODO.

**Pull request merged**